### PR TITLE
Task3 creci

### PR DESCRIPTION
**Pull request by creci**
Added xlgyro_fake to simulate xlgyro.

### CODE REVIEW

File: sbc-platform/src/pits/xlgyro_fake/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(cmake_for_gl LANGUAGES C)
```
Comment (anton-mir) [inline sbc-platform/src/pits/xlgyro_fake/CMakeLists.txt]: plz rename target, for example "xlgyro_fake_input"

Comment (anton-mir) [reply sbc-platform/src/pits/xlgyro_fake/CMakeLists.txt]: add also here cmake command to include curses library, this worked for me
project(cmake_for_gl LANGUAGES C)
find_package(Curses REQUIRED)
add_executable(cmake_for_gl main.c)
target_include_directories(cmake_for_gl PRIVATE ${CURSES_INCLUDE_DIR})
target_link_libraries(cmake_for_gl ${CURSES_LIBRARY})

File: .github/workflows/ccpp.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
...
+      run: sudo apt-get update && sudo apt-get install --yes autoconf automake autotools-dev build-essential doxygen freeglut3 freeglut3-dev gcc git libagg-dev libcairo2-dev libfreetype6-dev libmarisa-dev libpangocairo-1.0-0 libpango1.0-dev libprotobuf-dev libqt5svg5-dev libtool libxml2-dev make pkg-config protobuf-compiler subversion libgoogle-glog-dev qtbase5-dev libqt5gui5 qttools5-dev-tools qttools5-dev qtdeclarative5-dev qtlocation5-dev qtpositioning5-dev qt5-default qml-module-qtquick-* qml-module-qtgraphicaleffects qml-module-qtpositioning libboost-all-dev libcurl4-openssl-dev libssl-dev libcanberra-gtk* python-gps libjansson-dev libftdi-dev fluid libjsoncpp-dev libfltk1.3* fftw3 libgflags-dev
```
Comment (anton-mir) [inline .github/workflows/ccpp.yml:22]: need to add here to sudo apt-get install your library in order to pass CI test checks 
libncursesw5-dev or libncurses5-dev --- do you know what exact to use?

**Review by anton-mir — CHANGES_REQUESTED** [review]
See comments in code and also:
1) Plz change output when running utility:
```
Example: ./a.out /dev/ttyUSB0
When "/dev/ttyUSB0" - any port
```
add more readable like in other linux utils and change a.out to the name of util (xlgyro-fake)

2) When running
 ./cmake_for_gl /dev/ttyUSB0

 +----------------------------------+
 |        Serial Port Write         |
 +----------------------------------+
  Error! in Opening /dev/ttyUSB0

I have no USB0 attached, but it should work!
Maybe consider use virtual port emulator? Or how to run it?
Plz add more info then

File: sbc-platform/src/pits/xlgyro_fake/CMakeLists.txt (hunk 1 above)
Comment (anton-mir) [reply sbc-platform/src/pits/xlgyro_fake/CMakeLists.txt]: And also need to add 
`add_subdirectory(pits/xlgyro_fake)`
to sbc-platform/src/CMakeLists.txt

File: .github/workflows/ccpp.yml (hunk 2 above)
Comment (creci) [reply .github/workflows/ccpp.yml:22]: I do not know.

**Pull request merged**